frameworks/elastic/tests/test_shakedown.py

```diff
@@ -6,13 +8,15 @@
 import sdk_upgrade
 from tests.config import *
 
+log = logging.getLogger(__name__)
+
 FOLDERED_SERVICE_NAME = sdk_utils.get_foldered_name(PACKAGE_NAME)
```

Comment:
A great example of why not to ever do `from ____ import *` I have a hard time not fixing things if they are getting touched so I made those imports explicit but I won't complain if someone really wants that in another PR